<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(198, 149, 94)</string>
  </property>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 208, 133);
border-radius: 10px; 
font: 12pt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 208, 133);
color: black;
font: 25pt &quot;Bodoni MT&quot;;
</string>
   </property>
   <property name="text">
    <string> Daily Task Planner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>331</width>
     <height>251</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 208, 133);
font: 8pt &quot;Bodoni MT&quot;;
font-color: rgb(0, 0, 0)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 208, 133);
font: 8pt &quot;Open Sans&quot;;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>390</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 208, 133);
font: 8pt &quot;Open Sans&quot;;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>100</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt;
background-color:rgb(255, 208, 133);</string>
   </property>
   <property name="placeholderText">
    <string>Insert Text...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteTaskButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>390</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(172, 172, 172);
font: 8pt &quot;Open Sans&quot;;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="showTodayButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>331</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 208, 133);
font: 8pt &quot;Open Sans&quot;;
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string>Show Today</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt;</string>
   </property>
   <property name="text">
    <string>Remember to Save Changes after you have checked/unchecked a task.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
